services/contract_create.proto

```diff
@@ -1,13 +1,46 @@
+/**
+ * # Smart Contract Create
+ *
+ * Create a new smart contract.
+ *
+ * ## General Comments
+ *  - A smart contract normally enforces rules, so "the code is law".<br/>
+ *    For example, an ERC-20 contract prevents a transfer from being undone
+ *    without a signature by the recipient of the transfer. This characteristic
+ *    is generally true if the contract instance was created without a value
+ *    for the `adminKey` field. For some uses, however, it may be desirable to
+ *    create something like an ERC-20 contract that has a specific group of
+ *    trusted individuals who can act as a "supreme court" with the ability to
+ *    override the normal operation, when a sufficient number of them agree to
+ *    do so. If `adminKey` is set to a valid Key (which MAY be complex), then a
```

Comment:
It is quite possible that we may be deprecating the ability to set a cryptographic admin key for contracts in the near future.  If this is the case, then the only admin key allowed will be a contract key which allows for self management by the contract.

Reply:
That would definitely require a separate HIP, but good to be aware.